pom.xml

```diff
@@ -1053,6 +1117,19 @@
         <scope>test</scope>
       </dependency>
 
+      <dependency>
+        <groupId>org.apache.flink</groupId>
+        <artifactId>flink-test-utils_${scala.binary.version}</artifactId>
+        <version>${flink.version}</version>
+        <scope>test</scope>
```

Comment:
Where is this dependency be used ?

Reply:
https://github.com/apache/hudi/blob/master/hudi-client/hudi-flink-client/src/test/java/org/apache/hudi/testutils/HoodieFlinkClientTestHarness.java#L39